styles.css

```diff
@@ -729,4 +829,64 @@ header {
   .mobile-menu {
     display: none;
   }
+
+  /* Desktop version Pop up style */
+  .overlay-preview {
+    position: fixed;
+    opacity: 0;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    background: #f7f7f9;
+    backdrop-filter: blur(5px);
+    mix-blend-mode: multiply;
+    pointer-events: none;
+    transition: 200ms ease-in-out;
+  }
+
+  .overlay-preview.active {
+    opacity: 1;
+    pointer-events: all;
+  }
+
+  .preview .right-block {
+    display: grid;
+    grid-template-columns: 68% 32%;
+    grid-gap: 10px;
+    padding: 0;
+    margin: 31px 0 0 0;
+  }
```

Comment:
- [ ] Kindly consider using `fraction` for your `grid` columns instead of percentages. Currently, your design is overflowing on desktop screen size `(768px)` and above, this issue can be fixed by using `fraction`. You can set your `grid-template-column: 2fr 1fr;` such that your paragram text will take more space than the `buttons` and `tags` on right.
- [ ] Using `fraction ` as I have suggested above can help make your design more responsive across screen sizes and that will help create a good user experience. Take a look at the screenshot below for more details.

Figma:
<img width="714" alt="Screenshot 2023-04-20 at 21 27 42" src="https://user-images.githubusercontent.com/107192076/233491666-0a328554-4a99-47a1-9cdd-6928676975f4.png">

Your design:
<img width="863" alt="Screenshot 2023-04-20 at 21 24 19" src="https://user-images.githubusercontent.com/107192076/233491395-5896c2ec-1cc3-4b4d-862c-30f33bf0821e.png">